installer.sh

```diff
@@ -369,6 +369,8 @@ print_file_info "$BINUTIL_VERSION_NAME" "The GNU Binutils are a collection of bi
 print_tool_info "texinfo" 1
 print_tool_info "gcc" 1
 print_tool_info "build-essential" 1
+print_tool_info "gawk" 1
```

Comment:
We should start commenting from which other tool/module this is needed. Could you please add this here.
We are going to start with the other tools